.github/workflows/test.yaml

```diff
@@ -1,6 +1,24 @@
 name: Testing
 on: [push, pull_request]
 jobs:
+  lint:
+    name: ESLint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install Node v16
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
```

Comment:
I thiiiiiink you can cache npms deps here

Reply:
Hmm, I added it in, let see

Reply:
Looks like it worked 👍